dask_kubernetes/__init__.py

```diff
@@ -15,7 +15,13 @@
 from .helm import HelmCluster
 
 __all__ = ["HelmCluster", "KubeCluster"]
-__version__ = _version.get_versions()["version"]
+
+try:
+    from ._version import version as __version__  # noqa
+    from ._version import version_tuple as __version_tuple__  # noqa
+except ImportError:
+    __version__ = "0.0.0"
+    __version_tuple__ = (0, 0, 0)
```

Comment:
The `_version.py` file is no longer kept in version control and instead is generated when running `pip install -e .` or when packaging the distribution for PyPI/Conda Forge. So it's possible to clone the repo and never install it, so we catch the `ImportError` and use a null version so imports at least work.